src/main/perl/FileEditor.pm

```diff
@@ -58,13 +58,38 @@ sub _is_valid_source
     return -f $fn;
 }
 
+sub _is_reference_newer
+{
+    my ($self) = @_;
+    my $is_newer = 0;   # Assume false
+    if ( $self->_is_valid_source(*$self->{options}->{source}) ) {
+        if ( !$self->_is_valid_source(*$self->{filename}) || 
+             ((stat(*$self->{options}->{source}))[9] > (stat(*$self->{filename}))[9]) ) {
```

Comment:
i'm not sure about `.rpm-new`, but rpms maintain timestamps of files, so files added by rpms at time X might have older timestamps.
also CAF (via LC) allows to set the mtime, so this is might give some unwanted sideeffects.
(but i like the idea of the PR)